### ISSUE TITLE
UNKNOWN property warning when observers are built with string concatenation

### ISSUE THREAD

**Issue by ebidel** [PolymerLabs/polylint#63]
Perhaps because it is only script?

<img width="691" alt="screen shot 2015-11-10 at 6 01 15 pm" src="https://cloud.githubusercontent.com/assets/238208/11081842/13d2404a-87d5-11e5-9b6d-4970daa13200.png">

**Comment by davidmaxwaterman** [comment]
https://github.com/PolymerElements/iron-ajax/issues/151

**Comment by vinvol** [comment]
Hi,

It turns out the problem comes from the observers string concat with +.
```
    observers: [
      '_requestOptionsChanged(url, method, params.*, headers, contentType, ' +
          'body, sync, handleAs, jsonPrefix, withCredentials, timeout, auto)'
    ],
```

Hydrolysis uses ast-utils which doesn't support BinaryExpression ( string + string ).
See expressionToValue in ast-values:119

**Comment by davidmaxwaterman** [comment]
Confirming that joining those two lines in iron-ajax.html gets rid of the error. ES6 template string doesn't help.

NB, the title of this report misspells 'UNKNOWN'.

**Comment by cdata** [comment]
Even more topical now that we are running Polylint in CI. It would be nice if this could be fixed, even if that just means making the error message more meaningful.

https://travis-ci.org/PolymerElements/iron-form/builds/101731117
